LiteFarmOrg/LiteFarm
# LF-2383/notify user when task assigned to them has been abandoned

Ticket [LF-2383](https://lite-farm.atlassian.net/browse/LF-2383)

**Description:**

- When a task has been abandoned by another user (i.e. management or owner), the worker will be notified
- @AbdusShaikh's [PR](https://github.com/LiteFarmOrg/LiteFarm/pull/1888) has been brought back

**To Test:**

1. You will need at least one worker at your farm that you have access to.
2. Create a task and assign it to the worker from step 1.
3. Abandon the task from step 2.
4. Log in to the account of the worker you initially assigned the task to from step 1.
5. You will see two notifications (initial assignment notification and abandonment notification)
6. Run 'Task Notification Tests'

Abandoner's name in the notification will be the user that performed the abandonment action (step 3).
